src/modules/fs/api.c

```diff
@@ -20,22 +26,119 @@ int writeToFile(const char *filename, const char *data) {
     return 1;
 }
 
-char *readFromFile(const char *filename) {
-    FILE* f = fopen(filename, "rb");
+char *readFromFile(const char *filename)
+{
+    FILE *f = fopen(filename, "rb");
     char *str = malloc(1);
     int len = 1;
 
-    if (f == NULL) {
+    if (f == NULL)
+    {
         return NULL;
     }
 
-    while (!feof(f)) {
+    while (!feof(f))
+    {
         str = realloc(str, len + 1);
-        str[len - 1] = (char) fgetc(f);
+        str[len - 1] = (char)fgetc(f);
         len++;
     }
     str[len - 2] = '\0';
 
     fclose(f);
     return str;
 }
+
+int deleteFile(const char *filename)
+{
+    char command[256];
+    sprintf(command, "rm %s", filename);
+
+    if (system(command) == 0)
```

Comment:
Don't call system commands, what if they aren't present? see man page for `unlink(2)`